irohad/main/impl/consensus_init.cpp

```diff
@@ -55,27 +55,31 @@ namespace iroha {
       std::shared_ptr<consensus::yac::Yac> YacInit::createYac(
           std::string network_address,
           ClusterOrdering initial_order,
-          const keypair_t &keypair) {
+          const keypair_t &keypair,
+          std::chrono::milliseconds delay_milliseconds) {
         return Yac::create(
             YacVoteStorage(),
             createNetwork(std::move(network_address), initial_order.getPeers()),
             createCryptoProvider(keypair),
             createTimer(),
             initial_order,
-            delay_seconds_ * 1000);
+            delay_milliseconds.count());
```

Comment:
maybe change `Yac` (and other classes) to use `std::chrono::*` as well?

Reply:
This will definitely be for future work.